app/services/trainees/filter.rb

```diff
@@ -108,7 +108,11 @@ def provider(trainees, provider)
     def submission_ready(trainees, record_completion)
```

Comment:
Should we change this method name? It doesn't have anything to do with submission_ready any more, well, not in this class it doesn't.

Reply:
I've changed the name for that method now to record_completion which is in line with how we've name the other filters in that file